### PR TITLE
Update helper text on the modal to indicate usage on multi-page documents

### PR DESCRIPTION
**Pull request by metheos**
### Pull Request Description
Updates the text on the signature placement modal to better explain how to place a signature on any page.

### Related Issue
Issue Number: 2903

### Pull Request Type
Other: Text Change

### CODE REVIEW

File: src/Components/Request/VisibleElements.vue [hunk 1]
```diff
@@ -14,7 +14,7 @@
 			</h2>
 			<p>
 				<small>
-					{{ t('libresign', 'Select each signer to define their signature positions') }}
+					{{ t('libresign', 'Click the page to sign, then select a signer to set their signature position') }}
```
Comment (vitormattos) [inline src/Components/Request/VisibleElements.vue:17]: What you think about this suggestion?

```suggestion
					{{ t('libresign', 'Click the page to add visible signature, then select a signer to set their signature position') }}
```

Is possible to sign without visible signature. If you don't want to add visible signature, you only need to ignore this step and go ahead.

**Review by vitormattos — CHANGES_REQUESTED** [review]
Thanks a lot by your contribution!

Could you fix the DCO issue?

Follow the instructions:

https://github.com/LibreSign/libresign/pull/2915/checks?check_run_id=24313652582

**Review by metheos — COMMENTED** [review]
Looks good. I think I fixed the PR, DCO shows green 👍
I enabled 'always add sign-off' in my vscode editor's git

**Comment by vitormattos** [comment]
Workded fine now!
Thanks!
The problem now is about the commit message. We followed at LibreSign the same pattern to commit messages of Nextcloud server and other apps of Nextcloud organization.
https://www.conventionalcommits.org/
I will accept your PR and we need to think more about how to help contributors to don't have problems with this pattern. This is not a block at your PR.

**Comment by metheos** [comment]
Thanks. I would like to contribute more in the future to this and other nextcloud projects, so I'll try to follow those guidelines next time ⭐

**Pull request merged**